app/models/organization.rb

```diff
@@ -3,6 +3,11 @@ class Organization < ActiveRecord::Base
   has_many :members, through: :memberships, source: :user
   has_many :administrators, class_name: :User
   has_many :crosswalks
+  has_many :development_team_memberships
+  has_many :developments, through: :development_team_memberships 
+  #this should be scoped for unique, but our Postgres version cannot select distinct on tables with JSON data types.
```

Comment:
Comments should go above the relevant code block, have a space between `#` and text, and have no more than 70 characters per line.